src/EnergyPlus/WeatherManager.cc

```diff
@@ -191,13 +191,9 @@ namespace WeatherManager {
 
     Array2D_bool TodayIsRain;             // Rain indicator, true=rain NOLINT(cert-err58-cpp)
     Array2D_bool TodayIsSnow;             // Snow indicator, true=snow NOLINT(cert-err58-cpp)
-    Array2D<Real64> TodayRainAmount;      // ficitious indicator of Rain NOLINT(cert-err58-cpp)
-    Array2D<Real64> TodaySnowAmount;      // ficitious indicator of Snow NOLINT(cert-err58-cpp)
```

Comment:
Several arrays were unused, so removing them: `TodayRainAmount, TodaySnowAmount, TodayOutWetBulbTemp, TodayOutHumRat, TomorrowRainAmount, TomorrowSnowAmount.` If they become useful someday they can be added back.